public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
```

Comment:
Maybe provide a bit of yourself and not just `Member of EddieHub`?

Reply:
@Panquesito7 i will do that by today itself